--- a/dbus/com.system76.CosmicGreeter.conf	2026-04-27 16:07:58.970727085 +1000
+++ b/dbus/com.system76.CosmicGreeter.conf	2026-04-27 16:08:13.385058014 +1000
@@ -2,7 +2,7 @@
           "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
           "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
 <busconfig>
-    <policy group="cosmic-greeter">
+    <policy group="_cosmic-greeter">
         <allow send_destination="com.system76.CosmicGreeter"/>
         <allow receive_sender="com.system76.CosmicGreeter"/>
     </policy>
