diff --git a/data/start-cosmic b/data/start-cosmic
index c91f463..b4c0a03 100755
--- a/data/start-cosmic
+++ b/data/start-cosmic
@@ -1,30 +1,16 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
 
 # From: https://people.debian.org/~mpitt/systemd.conf-2016-graphical-session.pdf
 
-if command -v systemctl >/dev/null; then
-    # robustness: if the previous graphical session left some failed units,
-    # reset them so that they don't break this startup
-    for unit in $(systemctl --user --no-legend --state=failed --plain list-units | cut -f1 -d' '); do
-        partof="$(systemctl --user show -p PartOf --value "$unit")"
-        for target in cosmic-session.target graphical-session.target; do
-            if [ "$partof" = "$target" ]; then
-                systemctl --user reset-failed "$unit"
-                break
-            fi
-        done
-    done
-fi
-
 # use the user's preferred shell to acquire environment variables
 # see: https://github.com/pop-os/cosmic-session/issues/23
 if [ -n "${SHELL}" ]; then
     # --in-login-shell: our flag to indicate that we don't need to recurse any further
     if [ "${1}" != "--in-login-shell" ]; then
         # `exec -l`: like `login`, prefixes $SHELL with a hyphen to start a login shell
-        exec bash -c "exec -l '${SHELL}' -c '${0} --in-login-shell'"
+        exec sh -c "exec '${SHELL}' -c '${0} --in-login-shell'"
     fi
 fi
 
@@ -76,36 +62,6 @@ if [ -d "/run/user/$(id -u)/keyring" ]; then
     fi
 fi
 
-if command -v systemctl >/dev/null; then
-    # Import some variables that we explicitly want to have available
-    # in the user session.
-    systemctl --user import-environment XDG_SESSION_TYPE XDG_CURRENT_DESKTOP DCONF_PROFILE SSH_AUTH_SOCK
-
-    # For environment variables already imported into the user's
-    # session, if the value imported differs from the value in this
-    # environment, update it.
-    mapfile -t existing_env_vars < <(systemctl --user show-environment)
-    for env_var in "${existing_env_vars[@]}"; do
-        env_var_name="${env_var%%=*}"
-        env_var_value=${!env_var_name:-}
-
-        # Skip current iteration if the environment variable's value
-        # in the current envionment is unset.
-        if [[ -z "${env_var_value}" ]]; then
-            continue
-        fi
-
-        env_var_val_str_to_compare="${env_var_name}=${env_var_value}"
-        env_var_val_str_to_compare_ansi_c_quoted="${env_var_name}=\$'$(printf '%q' "${env_var_value}")'"
-        if [[ "${env_var}" == "${env_var_val_str_to_compare}" ]]; then
-            continue
-        elif [[ "${env_var}" == "${env_var_val_str_to_compare_ansi_c_quoted}" ]]; then
-            continue
-        fi
-        systemctl --user import-environment "${env_var_name}" ||:
-    done
-fi
-
 # Run cosmic-session
 if [[ -z "${DBUS_SESSION_BUS_ADDRESS}" ]]; then
     exec /usr/bin/dbus-run-session -- /usr/bin/cosmic-session
