diff --git a/src/subscriptions/polkit_agent_helper.rs b/src/subscriptions/polkit_agent_helper.rs
index 4a1fe66..e7bfaa5 100644
--- a/src/subscriptions/polkit_agent_helper.rs
+++ b/src/subscriptions/polkit_agent_helper.rs
@@ -63,7 +63,7 @@ struct AgentHelper {
 
 impl AgentHelper {
     async fn new(pw_name: &str, cookie: &str) -> io::Result<(Self, Responder)> {
-        let mut child = Command::new(HELPER_PATH.unwrap_or("/usr/libexec/polkit-agent-helper-1"))
+        let mut child = Command::new(HELPER_PATH.unwrap_or("/usr/lib/polkit-1/polkit-agent-helper-1"))
             .arg(pw_name)
             .stdin(Stdio::piped())
             .stdout(Stdio::piped())
